optax/losses/segmentation.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+import chex
+import jax
+import jax.numpy as jnp
+
+from optax.losses.classification import sigmoid_binary_cross_entropy
+
+def sigmoid_focal_loss(
+    logits:  chex.Array,
+    labels:  chex.Array,
+    alpha: Optional[float] = None,
+    gamma: float = 2,
+) ->  chex.Array:
+    """Compute a sigmoid focal loss.
+    Implementation of the focal loss as used https://arxiv.org/abs/1708.02002.
```

Comment:
please add references on its own docstring subsection, something like
```

References
 X et al. 2018, http:/...
```

See _src/alias.py for many examples

Reply:
Done.